### ISSUE TITLE
how about a skybox?

### ISSUE THREAD

**Issue by amir-arad** [wwwtyro/space-scene-2d#1]
could you add a feature to generate a [skybox](https://doc.babylonjs.com/divingDeeper/environment/skybox)?

**Comment by wwwtyro** [comment]
Yes, that's the next item on my todo, though I'll be putting it in a separate repo. I'll ping you here when I've got it up.

**Comment by amir-arad** [comment]
Thanks @wwwtyro !
